app/src/main/java/com/eventyay/organizer/core/sponsor/create/CreateSponsorFragment.java

```diff
@@ -66,6 +66,11 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         }
 
         binding.submit.setOnClickListener(view -> {
+
+            binding.form.name.setText(binding.form.name.getText().toString().trim());
+            binding.form.quantity.setText(binding.form.quantity.getText().toString().trim());
+            binding.form.price.setText(binding.form.price.getText().toString().trim());
```

Comment:
Just trim in the presenter/viewmodel

Reply:
But how can this be done since validation is being done through the fragment?

Reply:
Just add validateEmpty to the XML then

Reply:
It's already there but I think it doesn't consider the 'only spaces' case